src/UnitTests/LanguageServerClient/UnitTests.LanguageServerClient/PythonLanguageClientMetadata.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABILITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System.Collections.Generic;
+using Microsoft.VisualStudio.LanguageServer.Client;
+
+namespace UnitTests.LanguageServerClient {
+    internal class PythonLanguageClientMetadata : ILanguageClientMetadata {
```

Comment:
is this actually used in any way? this metadata is very specific to how VS works so outside of VS, is it used?